pkg/traceutil/trace_test.go

```diff
@@ -0,0 +1,28 @@
+package traceutil
+
+import (
+	"testing"
+)
+
+func TestTrace(t *testing.T) {
+	var (
+		op    = "Test"
+		steps = []string{"Step1, Step2"}
```

Comment:
Things we should test:
- A trace with zero steps
- Trace timing is what we expect
-  Log output is formatted how we expect

This might be a good opportunity to use a golang test table.